Plinic/search/searchViews/searchResult.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  searchResult.swift
+//  Plinic
+//
+//  Created by 유경덕 on 2022/09/07.
+//
+
+import SwiftUI
+
+struct searchResult: View {
+
+    var nickName : String
+    var playlistTitle : String
+    
+    var body: some View {
+        ZStack  {
+            Color.black
+                .ignoresSafeArea()
+            VStack(spacing:0) {
+            
+                HStack {
+                    VStack(alignment:.leading, spacing: 0){
+                        Text("유저")
+                            .foregroundColor(Color.white)
+                            .fontWeight(.bold)
+                            .font(.system(size: 25))
+                            .frame(width: 390, height: 20, alignment: .leading)
+                            .padding(.leading, 10.0)
+                        Rectangle()
+                                 .fill(Color.white)
+                                 .frame(width: 200, height: 1)
+                                 .padding(.top, 10.0)
+                                 .padding(.leading, 10.0)
+                        Text("상위 검색결과")
+                            .foregroundColor(Color.white)
+                            .fontWeight(.bold)
+                            .font(.system(size: 15))
+                            .frame(width: 390, height: 20, alignment: .leading)
+                            .padding([.top, .leading], 10.0)
+                        HStack(alignment: .bottom){
+                            HStack{
+                                VStack{
+                                    Circle()
+                                    .frame(width: 120, height: 120)
+                                    .foregroundColor(Color.white)
+                                    Text("\(nickName)")
+                                        .foregroundColor(Color.white)
+                                        .fontWeight(.bold)
+                                        .font(.system(size: 15))
+                                    
+                                }//VStack
+                                .padding(.leading, 10)
+                                .padding(.top, 20.0)
+                                userContents(infoTitle:"플레이리스트",infoCount:"000")
+                            }//HStack
+                            VStack{
+                                Circle()
+                                .frame(width: 80, height: 80)
+                                .foregroundColor(Color.white)
+                                Text("\(nickName)")
+                                    .foregroundColor(Color.white)
+                                    .fontWeight(.bold)
+                                    .font(.system(size: 15))
+                                
+                            }//VStack
+                            .padding(.leading, 10.0)
```

Comment:
이 부분 전체를 하나의 파일(userResult)로 만들어주고 최상단 결과인 뷰 파일(userTopResult)을 하나 더 만든 후에 
UserResult().scaleEffect(스케일 키울 float값)
을 활용하여 만드는게 더 좋을거 같음
또한
```suggestion
                            VStack{
                               Image(profileImg)
                                .resizeable()
                                .aspectRatio(contentMode: .fill)
                                .clipShape(Circle())
                                // 이미지를 동그란형태로 클립하여 백엔드에서 이미지 정보를 주었을 때 profileImg 변수로 동작할수 있게끔 수정 바람
                                Text("\(nickName)")
                                    .foregroundColor(Color.white)
                                    .fontWeight(.bold)
                                    .font(.system(size: 15))
                                
                            }//VStack
                            .padding(.leading, 10.0)
```